requirements.txt

```diff
@@ -46,3 +44,4 @@ haystack-ai>=2.8
 docling-core[chunking]>=2.10.0
 docling>=2.18.0
 sentence-transformers>=3.0.0
+bitsandbytes
```

Comment:
I believe it's not needed, and we don't import from the library directly. If there's some problem with other dependencies not tracking their reliance on bitsandbytes, then we should add a comment describing which dependency fails, ideally with a link to a bug report where this is reported upstream.